Tao_Current.txt

```diff
@@ -444,32 +507,36 @@ You can always tell a person who doesn't understand the IETF (or is intentionall
+    The templates and scripts from Martin Thomson, [https://github.com/martinthomson/i-d-template], might help you more easily write an Internet-Draft and automate the submission process. 
+
+Also, you should visit the IETF Tools web pages, [https://tools.ietf.org], where you'll find pointers to other tools that will automate some of your work for the IETF.
 
-Also, you should visit the IETF Tools web pages, http://tools.ietf.org, where you'll find pointers to other tools that will automate some of your work for the IETF.
 6.3.2 Filenames and Other Matters
 
-When you're ready to turn in your Internet-Draft, you submit it to http://datatracker.ietf.org/submit. The instructions on that web page will walk you through the needed steps, and there is also an email address there in case you need personalized help.
+When you're ready to turn in your Internet-Draft, you submit it to [https://datatracker.ietf.org/submit]. The instructions on that web page will walk you through the needed steps, and there is also an email address there in case you need personalized help.
 
 When you submit the first version of the draft, you also tell the draft administrator your proposed filename for the draft. If the draft is an official Working Group product, the name will start with "draft-ietf-" followed by the designation of the WG, followed by a descriptive word or two, followed by "00.txt".
 
-For example, a draft in the S/MIME WG about creating keys might be named "draft-ietf-smime-keying-00.txt". If it's not the product of a Working Group, the name will start with "draft-" and the last name of one of the authors followed by a descriptive word or two, followed by "00.txt". For example, a draft that someone named Smith wrote might be named "draft-smith-keying-00.txt". If a draft is an individual submission but relates to a particular Working Group, authors sometimes follow their name with the name of the Working Group, such as "draft-smith-smime-keying-00.txt". If you follow the naming guidelines given at http://www.ietf.org/ietf/1id-guidelines.txt, chances are quite good that your suggested filename will be fine.
+For example, a draft in the S/MIME WG about creating keys might be named "draft-ietf-smime-keying-00.txt". If it's not the product of a Working Group, the name will start with "draft-" and the last name of one of the authors followed by a descriptive word or two, followed by "00.txt". For example, a draft that someone named Smith wrote might be named "draft-smith-keying-00.txt". If a draft is an individual submission but relates to a particular Working Group, authors sometimes follow their name with the name of the Working Group, such as "draft-smith-smime-keying-00.txt". If you follow the naming guidelines given at [https://www.ietf.org/ietf/1id-guidelines.txt], chances are quite good that your suggested filename will be fine.
```

Comment:
Note that I sometimes see draft-foo where foo is not a last name. E.g., draft-ymbk

Reply:
Not sure what to do with this.

Reply:
Maybe:
If it's not the product of a Working Group, the name will start with "draft-" but is not followed by "ietf-". Often the last name of one of the authors (or some other identifier) is followed by a descriptive word or two, followed by "00.txt".

Reply:
Accepted and changed